### PR TITLE
feat: update promo

### PR DESCRIPTION
**Pull request by gusliedke**
Type: patch

Fixes #264

### CODE REVIEW

File: src/components/Molecules/Promo/Promo.js [hunk 1]
```diff
@@ -5,53 +5,50 @@ import Picture from '../../Atoms/Picture/Picture';
 import {
   Container,
   NudgeBottom,
```
Comment (LouiaR) [inline src/components/Molecules/Promo/Promo.js:7]: Did you try clip-path ?

Comment (gusliedke) [reply src/components/Molecules/Promo/Promo.js:7]: No, clip-path is interesting .. looks like it has good support .. I will try it out!

Comment (LouiaR) [reply src/components/Molecules/Promo/Promo.js:7]: Can you update font family of CTA as well

**Pull request merged**